### DISCUSSION TITLE
Still under development? Could we get an update?

### DISCUSSION THREAD

**Discussion by RandomGgames** [mrshmllow/BetterRecipeBook#62] [General]
Hey Mrshmllow, how have you been?

I wanted to ask something. I noticed that this mod hasn't gotten any updates in a while, scrolling through some issues and pull requests I noticed you were having trouble updating the mod every update. I'm curious if you're still actively working on the mod or if it's become too much and you don't have enough time/passion anymore. I would love to just hear an update on it! I used this mod all the time and have grown to miss it as I've updated my client to 1.19+ versions.

In any case, I'm not trying to bug you, I think I as well as many in the community would just like an update on this project. I appreciate the hard work you've put in over the years of working on it and bringing this mod to the community.

**Comment by mrshmllow** [comment]
I would like to improve this mod. I made it over a Covid-idy summer break when I played a lot of minecraft with friends and I could just slam out big projects care free. I want it as much as you do trust me but i simply have so many other things and so little time these days

**Reply by RandomGgames** [reply]
I definitely get it, Covid made working on projects at home a lot easier. Now that things are mostly if not completely back on track, a lot of those projects gotta be put on the back end unfortunately due to regular higher priority stuff is back on the todo list.

I wish you the best of luck! Thanks for the speedy reply and have a good one. I hope to hear from you in the future!
